### PR TITLE
Configure the Intercom canvas to use text saved in database

### PR DESCRIPTION
**Pull request by ruthwickp**
The Cadet Canvas App in Intercom will use text saved by each company setting or the default text. 

I DID NOT provide an endpoint to configure the text and label since it is overkill. To modify it for a given company, we must go into the database and update the field. This should not be a big deal since only one user requested it. 

Thoughts?

### CODE REVIEW

**Comment by pranav7** [comment]
I think it might not be _that_ hard to allow the user to actually configure it themselves while setting up the app. For example, we can just add two new fields here: https://github.com/pranav7/cadet-app/blob/e74ab3843c7e965c1d999c834c2b136dd26ed513/app/controllers/intercom_controller.rb#L106-L133

**Comment by pranav7** [comment]
I also think since we are going to be working more closely with Intercom, it might make sense to think about the data modeling of saving intercom settings with first principles. For example, we have a lot of settings that just live on the `CompanySetting` table, we might have an opportunity to normalize that to ease future development?

Maybe we can have a quick catch-up to discuss this one tomorrow?

**Comment by ruthwickp** [comment]
> I also think since we are going to be working more closely with Intercom, it might make sense to think about the data modeling of saving intercom settings with first principles. For example, we have a lot of settings that just live on the `CompanySetting` table, we might have an opportunity to normalize that to ease future development?
> 
> Maybe we can have a quick catch-up to discuss this one tomorrow?

Sounds good. Let's discuss that.

**Comment by ruthwickp** [comment]
Here is the form to edit the Intercom App fields.

Screen Shot 2021-04-29 at 10 43 33 AM

File: app/controllers/admin/integrations_controller.rb [hunk 1]
```diff
@@ -4,6 +4,18 @@ class Admin::IntegrationsController < Admin::AdminController
   def show
     @api_key = current_company.company_setting.api_key
     @intercom_access_token = current_company.company_setting.intercom_access_token
+    @company_setting = current_company.company_setting
+    @canvas_label = 'Share ideas or feedback'
+    @canvas_text = 'Share Feedback'
+    @canvas_label = @company_setting.intercom_canvas_settings['canvas_label'] if @company_setting.intercom_canvas_settings && !@company_setting.intercom_canvas_settings['canvas_label'].blank?
```
Comment (pranav7) [inline app/controllers/admin/integrations_controller.rb:10]: `.present?` is what want here maybe? Do we only want to set it if it's present?

```suggestion
    @canvas_label = @company_setting.intercom_canvas_settings['canvas_label'] if @company_setting.intercom_canvas_settings && @company_setting.intercom_canvas_settings['canvas_label'].present?
```

**Comment by pranav7** [comment]
Looks good otherwise, we can test it out in production 👍

**Pull request merged**